src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -6790,6 +7007,250 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      x-alfresco-since: "7.4"
+      tags:
+        - tags
+      summary: Delete a tag
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.4 and newer versions.
+
+        Deletes the tag with **tagId**. This will cause the tag to be removed from all nodes.
+      operationId: deleteTag
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/tagIdParam'
+      responses:
+        '204':
+          description: Successful response
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user is not an administrator and so does not have permission to delete the tag.
+        '404':
+          description: |
+            **tagId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/category/{categoryId}':
+    get:
+      x-alfresco-since: "7.4"
+      tags:
+        - categories
+      summary: Get a category
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.4 and newer versions.
+
+        Get a specific tag with **categoryId**.
+      produces:
+        - application/json
+      operationId: getCategory
+      parameters:
+        - $ref: '#/parameters/categoryIdParam'
+        - $ref: '#/parameters/fieldsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/CategoryEntry'
+        '401':
+          description: Authentication failed
+        '404':
+          description: |
+            **categoryId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      x-alfresco-since: "7.4"
+      tags:
+        - categories
+      summary: Update a category
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.4 and newer versions.
+
+        Updates the category **categoryId**.
+      operationId: updateCategory
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/categoryIdParam'
+        - $ref: '#/parameters/fieldsParam'
+        - in: body
+          name: categoryBodyUpdate
+          description: The updated category
+          required: true
+          schema:
+            $ref: '#/definitions/CategoryBody'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/CategoryEntry'
+        '400':
+          description: |
+            Invalid parameter: **categoryBodyUpdate** is invalid
```

Comment:
Shouldn't the description be like: `**categoryId** is not a valid format or **categoryBodyUpdate** is invalid`